{"name": "test_run_invalid_api_wrapper", "status": "passed", "description": "Test that _run fails gracefully when `api_wrapper.run` is not implemented correctly.", "start": 1743504728113, "stop": 1743504728113, "uuid": "464a5d57-9caa-431d-ac2e-2c1d6b0f3b9a", "historyId": "0cfd6ac5aa753ad68889f8c0c1757507", "testCaseId": "0cfd6ac5aa753ad68889f8c0c1757507", "fullName": "src.tests.test_base_tool.TestBaseAction#test_run_invalid_api_wrapper", "labels": [{"name": "tag", "value": "negative"}, {"name": "tag", "value": "base"}, {"name": "tag", "value": "unit"}, {"name": "parentSuite", "value": "src.tests"}, {"name": "suite", "value": "test_base_tool"}, {"name": "subSuite", "value": "TestBaseAction"}, {"name": "host", "value": "fv-az1927-746"}, {"name": "thread", "value": "2344-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.tests.test_base_tool"}]}